tools/gstf_preparation/gstf_preparation.py

```diff
@@ -66,8 +66,10 @@ def create_tables(conn):
 
     cur.execute('''CREATE TABLE transcript (
         transcript_id VARCHAR PRIMARY KEY NOT NULL,
+        transcript_symbol VARCHAR,
         protein_id VARCHAR UNIQUE,
         protein_sequence VARCHAR,
+        biotype VARCHAR,
```

Comment:
This seems redundant since we are already filtering out non-protein_coding transcripts ( https://github.com/TGAC/earlham-galaxytools/pull/123 )

Reply:
Sometimes Biotype mentioned in Gene line rather than transcript line so to avoid that posibility

Reply:
I mean that's always going to be either `protein_coding` or NULL.

Reply:
Unless you want to keep also the non-protein coding genes and transcripts in the database, and only do the filtering when writing the (merged and filtered) output FASTA file.

Let me know!

Reply:
Decided to keep also the non-protein coding genes and transcripts in the database.